src/test/config/jira/service-expose-node.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ${SERVICE_NAME}
+spec:
+  type: ClusterIP
+  ports:
+    - port: ${PORT_PRODUCT_SERVICE}
+      protocol: TCP
+      targetPort: ${PORT_PRODUCT_CONTAINER}
+  selector:
+    statefulset.kubernetes.io/pod-name: ${SERVICE_NAME}
```

Comment:
this is a little misleading.... the name of this backdoor service is the same as the name of the pod it's backdoor-ing, so maybe this var should be `POD_NAME`

Reply:
Renamed to `$podName`.